ale/transformation.py

```diff
@@ -380,7 +380,7 @@ def extract_exact_ck_times(observStart, observEnd, targetFrame):
 
         return times
 
-    def compute_time_dependent_rotiations(self, frames, times, time_bias):
+    def compute_time_dependent_rotations(self, frames, times, time_bias):
```

Comment:
This is an internal function, used only in that file.

Reply:
I agree that it is an internally used function but a user could access the function if they wanted to and compute other frame rotations on a framechain object

Reply:
This also came up in the usgscsm repository; with no consensus, as I see.

There should be a clear policy, as to what parts of a library are the user-facing API, which must be guaranteed to be kept fixed, or at least not modified without a good reason, and which part is internal implementation. Constraining all internal functions is not possible.

Reply:
At least in ISIS, it's any function that a user has access to. Since that is C/C++ code that is more enforceable. Typically in python a "private" function is defined with a leading `_` so that is something we could do in our python libraries. Similarly we should make functions private in USGSCSM that should not be public facing API